src/libs/protocols/build.rs

```diff
@@ -229,6 +229,8 @@ fn real_main() -> Result<(), std::io::Error> {
         false,
     )?;
 
+    codegen("src", &["protos/cri-api/api.proto"], false)?;
```

Comment:
Any reason for not passing it to the existing invocation of `codegen()` ?

Reply:
Oh, I just want to group these codegens according to different uses to make it clearer for subsequent readers.